src/shoobx/rml2odt/directive.py

```diff
@@ -14,15 +14,14 @@
 """RML Directive Implementations
 """
 import lxml
-
 from z3c.rml import directive
 
 
 class NotImplementedDirective(directive.RMLDirective):
 
     def process(self):
         raise NotImplementedError(
-            u'Directive not implemented: %s' % self.element.tag)
+            'Directive not implemented: %s' % self.element.tag)
```

Comment:
format string or just addition?

Reply:
i'm not going back in and doing this again. It's just what pyupgrade does.